Documentation edits (#330)
ansys/pysystem-coupling: doc/source/users_guide/index.rst

Exiting
-------

> When you are finished with a PySystemCoupling session, we recommend ending it cleanly using the

Reviewer: ```suggestion
When you are finished with a PySystemCoupling session, you should end it cleanly using the
```

> ``exit()`` method. If you do not do this, PySystemCoupling will still attempt to clean up any active server sessions

Reviewer: ```suggestion
``exit()`` method. If you do not do this, PySystemCoupling still attempts to clean up any active server sessions
```
Use present tense.
